src/app/App.tsx

```diff
@@ -443,6 +471,42 @@ class App extends AsyncComponent<RouteComponentProps, AppState> {
 
         document.title = title;
     }
+
+    /**
+     * Refresh the token one minute before it expires.
+     */
+    private validateTokenPeriodically() {
+        if (this._tokenExpiryTimer !== undefined) {
+            clearInterval(this._tokenExpiryTimer);
+            this._tokenExpiryTimer = undefined;
+        }
+        const jwt = this._storageService.load<string>("dashboard-jwt");
+        const expiryTime = this.getTokenExpiry(jwt);
```

Comment:
This is more like an `expiryTimestamp`, than just time. In the sense that it carries not just the hours minues but also date, so like a dateTime.